.eslintrc

```diff
@@ -16,6 +16,7 @@
 			}
 		],
 		"jsx-a11y/anchor-is-valid": "off",
+		"jsx-a11y/alt-text": "off",
```

Comment:
I do think the rule is important, but eslint doesn't seem to understand I'm including the `alt` in the object...